api/checks/webhook.go

```diff
@@ -267,7 +264,9 @@ func createWPTCheckSuite(ctx context.Context, appID, installationID int64, sha s
 		HeadSHA: sha,
 	}
 	suite, _, err := client.Checks.CreateCheckSuite(ctx, wptRepoOwner, wptRepoName, opts)
-	if err == nil && suite != nil {
+	if err != nil {
+		log.Errorf("Failed to create GitHub check suite: %s", err.Error())
+	} else if suite != nil {
```

Comment:
nit: I'm assuming you can't have both err and suite be set at the same time? Although removing the "else" and just having two separate "ifs" might be more resilient if that assumption ever changes.